pkg/storage/unified/blob/types.go

```diff
@@ -0,0 +1,19 @@
+package blob
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+type PublicBlobStore interface {
```

Comment:
TODO!  We can follow up in an new PR to make this something we are more comfortable running in cloud.  Unified storage already has a bunch configured to handle object storage but I don't think we should use it directly -- this is a write only interface that should also replace our current PDF rendering store